apps/st2-history/history-popup.component.js

```diff
@@ -99,6 +99,8 @@ export default class HistoryPopup extends React.Component {
     return (
       <div className="st2-rerun">
         <Popup title="Rerun an execution" onCancel={onCancel} data-test="rerun_popup">
+          <h4 style={{ padding:'20px 20px 0' }}>The input values from the previous run are applied by default and displayed in black. The original default values of the action parameters are displayed in grey.</h4>
```

Comment:
@pradnya-orchestral Can you make this normal text, larger text but not bold and closer to the title?  Thanks